macros/src/main/scala/me/rexim/morganey/meta/package.scala

```diff
@@ -0,0 +1,19 @@
+package me.rexim.morganey
+
+import scala.language.experimental.macros
+
+package object meta extends AnyRef
+  with DefaultLiftableInstances
+  with DefaultUnliftableInstances {
+
+  private[meta] trait Quasiquotation {
+    def apply[T](args: T*): Any = macro QuotationMacro.quote
+    def unapply(arg: Any): Any = macro QuotationMacro.unquote
+  }
+
+  implicit class Quasiquote(ctx: StringContext) {
+    object m extends Quasiquotation
+    object lc extends Quasiquotation
```

Comment:
I'm not against using `m` or `lc` or both simultaneously. I was just curious way do we have two interpolators that do the same thing. Now I see what they mean. I'm ok with providing them both to the user as available options.